src/main/java/at/hannibal2/skyhanni/features/inventory/SuperCraftFeatures.kt

```diff
@@ -0,0 +1,31 @@
+package at.hannibal2.skyhanni.features.inventory
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.api.GetFromSackAPI
+import at.hannibal2.skyhanni.events.LorenzChatEvent
+import at.hannibal2.skyhanni.utils.DelayedRun
+import at.hannibal2.skyhanni.utils.NEUInternalName
+import at.hannibal2.skyhanni.utils.PrimitiveItemStack
+import at.hannibal2.skyhanni.utils.StringUtils.matchMatcher
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class SuperCraftFeatures {
+    private val craftedPattern by RepoPattern.pattern(
+        "inventory.supercrafting.craft",
+        "§eYou Supercrafted §r§r§r§.(?<item>[^§]+)(?:§r§8x(?<amount>\\d+))?§r§e!"
+    )
+    private val config get() = SkyHanniMod.feature.inventory.gfs
+
+    @SubscribeEvent
+    fun onChat(event: LorenzChatEvent) {
+        if (!config.superCraftGFS) return
+        val (internalName, amount) = craftedPattern.matchMatcher(event.message) {
+            NEUInternalName.fromItemName(this.group(1)) to (this.group(2)?.toInt() ?: 1)
```

Comment:
now just use the named groups :)
```suggestion
            NEUInternalName.fromItemName(this.group("item")) to (this.group("amount")?.toInt() ?: 1)
```

Reply:
lol